src/clims/api/endpoints/work_definition_details.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from sentry.api.base import Endpoint, DEFAULT_AUTHENTICATION
+from clims.utils import single_or_default
+from clims.api.serializers.models.work_definition import WorkDefinitionSerializer
+from clims.api.serializers.models.workunit import WorkUnitSerializer
+
+
+class WorkDefinitionDetailsEndpoint(Endpoint):
```

Comment:
Which is used?!

Reply:
Here, things get messy however. Exactly what fields should be included in work definition, workbatch and work configuration respectively? Take properties in work-configuration for example, (which is step udfs in clarity). Work configuration is at present thought of to be a field within a workbatch, so that all extensible properties may be lumped into one single field. In principle though, as far as my understanding goes, these fields could equally well be included in work definition? Maybe work-defnition and work-configuration is really the same concept?

Reply:
There is another thing that came into my mind here. I make comments about the endpoints, but I think it's a slight difference between the endpoints and the domain model. In my mind, endpoints is much more liable to adjust to the demands at the UI, while the domain models aims for solving domain logic (do we agree on that?).

Reply:
Yes, I agree.